top/functions/gqa.py

```diff
@@ -17,7 +31,21 @@ def forward(ctx, Q, K, V, fwd_op, bwd_op):
         return O
 
     @staticmethod
-    def backward(ctx, dO):
+    def backward(ctx,
```

Comment:
Is this the common practice for functions that do not provide a backward implementation by default? We should verify this.